telemetry_core/src/main/java/com/newrelic/telemetry/spans/SpanBatchSenderBuilder.java

```diff
@@ -21,6 +21,7 @@
 
 public class SpanBatchSenderBuilder {
   private static final String spansPath = "/trace/v1";
+  private static final String DEFAULT_TRACE_URL = "https://trace-api.newrelic.com/";
```

Comment:
I pulled this thing out so that I could reference it in a javadoc without having to re-type it and risk this the field and doc getting out of sync.

Reply:
👍

Reply:
Solid idea!